### DISCUSSION TITLE
How to copy notes without configuration?

### DISCUSSION THREAD

**Discussion by ghost** [TriliumNext/Trilium#2446] [Q&A] [answered]
How to copy my notes without the configuation ?
when i download a new version of trilium i copy "backup-now.db" to Roaming\trilium-data as "document.db",
but it copt the old settings with it. I just want my notes with the fresh install

**Comment by zadam** [comment, marked as answer]
Hi, you can export the notes from your old installation into a ZIP file and the import into the new installation. That copies only notes (without revision history though).
![image](https://user-images.githubusercontent.com/617641/146271492-795c10da-8345-45e6-90e7-27e74813e0fb.png)
